tests/cockpit-tasks-centosci.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ReplicationController
```

Comment:
For my similar PR #206 it complained that I need to set `apiVersion: v1` here as well. Apparently this wasn't necessary at the time when we rolled this out on our bos cluster, but CentOS CI's version needs it.

That's the kind of fine-tuning that you can do when actually rolling this out. Try without, and if it complains about something like

    error: unable to get type info from the object "*unstructured.Unstructured": Object 'apiVersion' is missing in 'object has no apiVersion field'

then add it here.

Reply:
Hm centos ci does need the api version, it won't create one otherwise.